crates/bevy-inspector-egui/Cargo.toml

```diff
@@ -8,39 +8,55 @@ license = "MIT OR Apache-2.0"
 description = "Inspector plugin for the bevy game engine"
 documentation = "https://docs.rs/bevy-inspector-egui"
 keywords = ["bevy", "inspector", "egui", "editor", "reflect"]
-categories = ["game-development", "gui", "visualization", "development-tools::procedural-macro-helpers"]
+categories = [
+    "game-development",
+    "gui",
+    "visualization",
+    "development-tools::procedural-macro-helpers",
+]
 
 [features]
 default = ["bevy_pbr"]
 highlight_changes = []
 
 [dependencies]
-bevy-inspector-egui-derive = { version = "0.18.0", path = "../bevy-inspector-egui-derive" }
-bevy_app = { version = "0.10" }
-bevy_asset = { version = "0.10" }
-bevy_core = { version = "0.10" }
-bevy_ecs = { version = "0.10" }
-bevy_hierarchy = { version = "0.10" }
-bevy_log = { version = "0.10" }
-bevy_math = { version = "0.10" }
-bevy_reflect = { version = "0.10" }
-bevy_utils = { version = "0.10" }
-bevy_window = { version = "0.10" }
-
-bevy_core_pipeline = { version = "0.10" }
-bevy_pbr = { version = "0.10", optional = true }
-bevy_render = { version = "0.10" }
+bevy-inspector-egui-derive = { version = "0.18.1", path = "../bevy-inspector-egui-derive" }
+bevy_app = { version = "0.11" }
+bevy_asset = { version = "0.11" }
+bevy_core = { version = "0.11" }
+bevy_ecs = { version = "0.11" }
+bevy_hierarchy = { version = "0.11" }
+bevy_log = { version = "0.11" }
+bevy_math = { version = "0.11" }
+bevy_reflect = { version = "0.11" }
+bevy_utils = { version = "0.11" }
+bevy_window = { version = "0.11" }
+
+bevy_core_pipeline = { version = "0.11" }
+bevy_pbr = { version = "0.11", optional = true }
+bevy_render = { version = "0.11" }
 
 egui = "0.21"
-bevy_egui = "0.20"
+bevy_egui = { git = "https://github.com/mvlabat/bevy_egui.git", branch = "main" }
```

Comment:
Though pointing to official crates.io backed github repository, need to keep this PR open until we can switch to a released version.

Reply:
https://github.com/jakobhellermann/bevy-inspector-egui/pull/141/commits/3c0dac7402edb0a4a20e8881323334c669a9343f now points to `bevy_egui` `0.21`.

I have run the automated tests and did a quick run of all examples manually.